CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+## [1.10](https://github.com/brave/brave-ios/releases/tag/v1.10)
+
+ - Implemented new address bar layout. ([#1134](https://github.com/brave/brave-ios/issues/1134))
+ - Implemented new main menu layout. ([#1130](https://github.com/brave/brave-ios/issues/1130))
+ - Added ability to "Find in page" using the URL bar. ([#1019](https://github.com/brave/brave-ios/issues/1019))
+ - Added 1Password activity in share sheet. ([#948](https://github.com/brave/brave-ios/issues/948))
+ - Added image titles when using long-press on images. ([#851](https://github.com/brave/brave-ios/issues/851))
+ - Added privacy warning on Brave Sync code page. ([#1225](https://github.com/brave/brave-ios/issues/1225))
+ - Improved swipe gesture on bookmark hanger. ([#953](https://github.com/brave/brave-ios/issues/953)) 
+ - Improved scrolling issue on websites with sticky headers and footers. ([#631](https://github.com/brave/brave-ios/issues/631))
+ - Improved icon resolution for favourites. ([#463](https://github.com/brave/brave-ios/issues/463))
+ - Fixed several random crashes in certain conditions. ([#1120](https://github.com/brave/brave-ios/issues/1120))
+ - Fixed "Block-all-cookies" toggle behaviour. ([#897](https://github.com/brave/brave-ios/issues/897))
+ - Fixed not being able to enter passcode after lockout timer. ([#938](https://github.com/brave/brave-ios/issues/938))
```

Comment:
@srirambv did this one make it in as #1203 didn't? I'm not sure if those two fixes were fixed at the same time.

Reply:
Both are different issues and different fixes. #1203 fix is part of 1.11 now